app/bundles/CoreBundle/Resources/views/Offline/offline.html.twig

```diff
@@ -0,0 +1,71 @@
+{% if inline is not defined or inline is empty %}
```

Comment:
Looking at what values the `inline` var can have I can see it can be either `null` or `false`. So this condition will always be true.

Reply:
Removed in the last commit